enrolment-ui/Dockerfile

```diff
@@ -1,22 +1,8 @@
-# Copyright 2017(c) The Ontario Institute for Cancer Research. All rights reserved.
+# Copyright 2020(c) The Ontario Institute for Cancer Research. All rights reserved.
 
-FROM ubuntu:16.04
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+FROM node AS prebuilder
```

Comment:
we don't need a whole ubuntu distro for this. Will improve it further at a separate ticket to not carry the whole Node image either, just the built files which are what nginx uses.